src/Workspaces/Core/Portable/Workspace/Solution/AssetHint.cs

```diff
@@ -13,19 +13,32 @@ namespace Microsoft.CodeAnalysis.Serialization;
 [DataContract]
 internal readonly struct AssetHint
 {
-    public static readonly AssetHint None = default;
+    /// <summary>
+    /// Instance that will only look up solution-level data when searching for checksums.
+    /// </summary>
+    public static AssetHint SolutionOnly = default;
```

Comment:
a bunch of real product code was moved from None to this to indicate they just want top level info.